system-int-test/test/cmr/system_int_test/search/autocomplete_collection_facet_search_test.clj

```diff
@@ -0,0 +1,81 @@
+(ns cmr.system-int-test.search.autocomplete-collection-facet-search-test
+  "Integration tests for autocomplete collection search facets"
+  (:require
+    [clojure.test :refer :all]
+    [cheshire.core :as json]
+    [clojurewerkz.elastisch.rest :as esr]
+    [cmr.common.util :as util :refer [are3]]
+    [cmr.system-int-test.data2.core :as d]
+    [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+    [cmr.system-int-test.utils.ingest-util :as ingest]
+    [cmr.system-int-test.utils.index-util :as index]
+    [cmr.system-int-test.utils.search-util :as search]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(deftest autocomplete-suggest-test
+  (let [conn (esr/connect "http://localhost:9210")
+        _ (esr/post conn "http://localhost:9210/1_autocomplete/suggestion"
+                    {:headers {:Content-Type "application/json"}
+                     :body {:value "foo" :type "instrument"}})]
+    (index/wait-until-indexed)
+
+    (testing "returned result form test"
+             (let [response (search/get-autocomplete-suggestions "foo")
+                   body (:body response)
+                   data (json/parse-string body true)]
+               (is (contains? data :query))
+               (is (contains? data :results))
+               (is (contains? (:results data) :items))))
+
+    (testing "missing query param response test"
+             (is (thrown? clojure.lang.ExceptionInfo
```

Comment:
add `throw-exceptions false` to your request and do your test based off status code received.

Reply:
Agreed. We need to test the final response and status code which should not be 500.